discovery-provider/scripts/start.sh

```diff
@@ -70,17 +68,23 @@ if [ "$audius_db_run_migrations" != false ]; then
 fi
 
 if [[ "$dev" == "true" ]]; then
-    ./scripts/dev-server.sh 2>&1 | tee >(cat >&2) server.log | logger &
-    if [[ "$audius_no_workers" != "true" ]] && [[ "$audius_no_workers" != "1" ]]; then
-        celery -A src.worker.celery worker --loglevel info 2>&1 | tee >(cat >&2) worker.log | logger &
+    ./scripts/dev-server.sh 2>&1 | tee >(logger -t server) server.log &
+    if [[ "$audius_only_web_server" != "true" ]]; then
+        if [[ "$audius_no_workers" != "true" ]] && [[ "$audius_no_workers" != "1" ]]; then
+            celery -A src.worker.celery worker --loglevel info 2>&1 | tee >(logger -t worker) worker.log &
+        fi
+        celery -A src.worker.celery beat --loglevel info 2>&1 | tee >(logger -t beat) beat.log &
     fi
-    celery -A src.worker.celery beat --loglevel info 2>&1 | tee >(cat >&2) beat.log | logger &
 else
-    ./scripts/prod-server.sh 2>&1 | tee >(cat >&2) | logger &
-    if [[ "$audius_no_workers" != "true" ]] && [[ "$audius_no_workers" != "1" ]]; then
-        celery -A src.worker.celery worker --loglevel info 2>&1 | tee >(cat >&2) | logger &
+    ./scripts/prod-server.sh 2>&1 | tee >(logger -t server) &
+    if [[ "$audius_only_web_server" != "true" ]]; then
+        if [[ "$audius_no_workers" != "true" ]] && [[ "$audius_no_workers" != "1" ]]; then
```

Comment:
i think we could probably consolidate the only web server and no worker flags into one thing since they do the same thing? the web-server.yml file could set no workers to true

Reply:
yes, sounds good, I'll make these changes now